src/View/Helper/PaginatorHelper.php

```diff
@@ -1169,4 +1170,40 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown select for pagination limit
```

Comment:
Might be good to mention that this will also generate the wrapping form as I am sure someone will put this inside an already open search form.